.travis.yml

```diff
@@ -54,3 +56,7 @@ script:
   - cd docs && make html && cd ..
   # Make sure the library installs.
   - python setup.py install
+  # Switch back to latest setuptools
+  - pip --upgrade setuptools pip
```

Comment:
I'd move this to the beginning of all the installs. Maybe right after: `- sudo apt-get update`